### ISSUE TITLE
[Feature Request] Multiple logins

### ISSUE THREAD

**Issue by MakerTim** [pyouroboros/ouroboros#371]
TLDR: i need multiple login's, for each docker repo one...

I pull from 2 diffrent docker repo's
one hosted on a private gitlab and from docker.io
Now when it tries to login on gitlab all is fine, but when it tries to login to docker.io it fails (like it should, the credentials of my private gitlab are not matching to my details on docker.io)

But now i have to run 2 ouroboro's instances, one for each repo im pulling from
could there be a change in command, or something for multiple login details for difrent repo's
or pull the details from the docker socket, containrrr/watchtower used to do that

**Comment by circa10a** [comment]
Have you tried [mounting in your config.json](https://github.com/pyouroboros/ouroboros/wiki/Private-Registries#mount-configjson-multiple-registry-credentials) ?
